ca_tools/launch/keyboard_teleop.launch

```diff
@@ -1,3 +1,5 @@
+<!-- USE MOVE_ROBOT VARIABLE TO SELECT WHICH ROBOT TO CONTROL, DEFAULT=1 -->
```

Comment:
This comment is not necessary anymore. Or, at least, it's outdated.

Reply:
Removed